.gitignore

```diff
@@ -17,3 +17,4 @@ datasets/
 /topicalguide/settings.py
 /working/
 tg_models.png
+/tools/stemmer/
```

Comment:
I don't think we want to have git totally ignore the stemmer directory. We do want it to ignore the Snowball and stemmer binaries inside that directory, though (users should build this, as directed in the README).